cfg/1.11/config.yaml

```diff
@@ -1,2 +1,15 @@
 ---
 ## Version-specific settings that override the values in cfg/config.yaml
+node:
```

Comment:
Are these really specific to 1.11, or should they be moved to `cfg/config.yaml`. Iirc these values were originally added to match the settings on EKS.